src/css/components/_transient-button.scss

```diff
@@ -0,0 +1,62 @@
+.video-js .vjs-transient-button {
+  position: absolute;
+  height: 3em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(50, 50, 50, 0.5);
+  cursor: pointer;
+  opacity: 1;
+  transition: opacity 1s;
+}
+
+.video-js:not(.vjs-has-started) .vjs-transient-button {
+  display: none;
+}
+
+.video-js .vjs-transient-button span {
+  padding: 0 0.5em;
+}
+
+.video-js .vjs-transient-button.left {
+  left: 5px;
+}
+
+.video-js .vjs-transient-button.right {
+  right: 5px;
+}
+
+.video-js .vjs-transient-button.top {
+  top: 5px;
+}
+
+.video-js .vjs-transient-button.bottom {
+  bottom: 35px;
+}
```

Comment:
Using em instead of px allows greater flexibility when changing player size. What do you think ?

With PX
[Screencast from 06. 03. 24 15:56:09.webm](https://github.com/videojs/video.js/assets/34163393/bcfc6c89-c0cf-48a2-b42e-868dab4003dc)

With EM
[Screencast from 06. 03. 24 15:57:20.webm](https://github.com/videojs/video.js/assets/34163393/bcda9e95-8855-4b24-b905-e923ff09846c)